### ISSUE TITLE
Deleting link list doesn't remove it from the grid

### ISSUE THREAD

**Issue by Woland2k** [VirtoCommerce/vc-platform#828]
Since list is not removed, trying to delete it again generates 500 error message:

![image](https://cloud.githubusercontent.com/assets/1566470/24489517/ceaa8dc4-14d2-11e7-8c5f-7bfaab50c07f.png)

**Comment by Andrew-Orlov** [comment]
There is an another issue with link lists: after creating a new link list it is not displayed in the list. It will appear only after reloading a page or some another action that will refresh this list.
